### PR TITLE
Make difficulty table margins equal

### PR DESCRIPTION
**Pull request by qwrtln**
Only German and Ukrainian required adjustments, all the other languages looked good. See the side margins of the original, which are equal now:

en-5

### CODE REVIEW

**Comment by DarthGandalf** [comment]
How did you come up with the new numbers?

How to do the numbers for new tables if we add any?

**Review by DarthGandalf — APPROVED** [review]

**Comment by qwrtln** [comment]
The answer to both questions is: trial and error.

**Comment by DerSchlupf** [comment]
The 4th Variante needs two lines in the german Version but that's the only difference, isnt it? And because of this the devil is smaller, right. It could be difficult to short this line more. It was also Tricky in the previous table Version. But i will search for shorter wordings in the future.

**Comment by qwrtln** [comment]
Yes, but it still fits on the page, so no problem.

**Review by Heegu-sama — APPROVED** [review]
